SPECS/mstflint/mstflint.spec

```
@@ -0,0 +1,298 @@
+%global __remake_config 1
+Summary:        Mellanox firmware burning tool
+Name:           mstflint
+Version:        4.21.0
+Release:        2%{?dist}
+License:        GPLv2+ OR BSD
```

This seems to be incorrect. Since the `COPYING` file in the source lists a specific GPL version and doesn't use the "or later" language, we shouldn't have the `+` attached to `GPLv2`.

```suggestion
License:        GPLv2 OR BSD
```

---

Done! Thank you for the catch!